src/containers/audio-rewards-page/components/modals/RewardsModals.tsx

```diff
@@ -1,18 +1,29 @@
-import React from 'react'
+import React, { Suspense, useEffect } from 'react'
 
 import { isMobile } from 'utils/clientUtil'
+import lazyWithPreload from 'utils/lazyWithPreload'
 
 import TopAPIModal from './TopAPI'
 import TransferAudioMobileDrawer from './TransferAudioMobileDrawer'
 import TrendingRewardsModal from './TrendingRewards'
 import VerifiedUpload from './VerifiedUpload'
 
+const HCaptchaModal = lazyWithPreload(() => import('./HCaptchaModal'))
```

Comment:
i don't actually think we need to preload this (yet) since we're not going to turn it on until there's evidence to? but should be fine either way

Reply:
> i don't actually think we need to preload this (yet) since we're not going to turn it on until there's evidence to? but should be fine either way

sorry somewhat whiplash from my part

Reply:
all good, will make the change